### PR TITLE
fix(properties): create property when undefined/void

### PR DESCRIPTION
**Pull request by Pmyl**

### CODE REVIEW

File: src/transformer/descriptor/mock/mockProperties.ts [hunk 1]
```diff
@@ -44,6 +44,10 @@ export function GetMockPropertiesFromDeclarations(
         return false;
       }
 
+      if (member.questionToken) {
```
Comment (uittorio) [inline src/transformer/descriptor/mock/mockProperties.ts:47]: This is cool. I think we can safely remove the code below from file propertySignature.ts

```
if (node.type) {
    if (node.questionToken) { // remove this if
      return GetUndefinedDescriptor();
    }

    return GetDescriptor(node.type, scope);
  }
```

What do you think?

File: test/playground/playground.test.ts [hunk 2]
```diff
@@ -10,9 +10,14 @@ import { createMock } from 'ts-auto-mock';
```
Comment (uittorio) [inline test/playground/playground.test.ts:10]: revert this one if you don't mind.

**Review by uittorio — CHANGES_REQUESTED** [review]
One comment!! Nice one!

File: src/transformer/descriptor/mock/mockProperties.ts (hunk 1 above)
Comment (Pmyl) [reply src/transformer/descriptor/mock/mockProperties.ts:47]: it makes sense, we exclude it early, it should never get to that point if it has the question token, I'll remove it later today

Comment (Pmyl) [reply src/transformer/descriptor/mock/mockProperties.ts:47]: done

File: test/playground/playground.test.ts (hunk 2 above)
Comment (Pmyl) [reply test/playground/playground.test.ts]: done

**Review by uittorio — APPROVED** [review]

**Pull request merged**